docker-compose.yml

```diff
@@ -1,27 +1,95 @@
-version: '3'
+version: '3.7'
 
 services:
   redis:
-    image: redis:4-alpine@sha256:18e31fe73a142c094fedbcdac27973233f82e5c9ef7b10386991deffd47caded
+    image: redis:4
     ports:
       - "6379:6379"
+
   mongo:
-    image: mongo:3.6.9-stretch@sha256:8b3c1f484805b85ce4b539fc5d7cda8824fff6fb91431ca2f03525d73878ad9a
+    image: mongo:3.6
     ports:
       - "27017:27017"
+
   rabbitmq:
-    image: rabbitmq:3.8.3-management-alpine@sha256:3459d0c7968053ee6e556a823c7eb3c13ac706ff1ce461dd62d259d74379cb87
+    image: rabbitmq:3.8.2-management
     ports:
       - "5672:5672"
       - "15672:15672"
     environment:
       RABBITMQ_DEFAULT_VHOST: db
+
   postgres:
-    image: postgres:11.7-alpine@sha256:785334846c220affdc714ecf06fab5006c96dbe2cef6ecd9860f21f330b5caeb
+    image: postgres:11
     ports:
       - "5432:5432"
     environment:
       LC_ALL: C.UTF-8
       POSTGRES_USER: admin
-      POSTGRES_PASSWORD: "${POSTGRES_PASSWORD}"
+      POSTGRES_PASSWORD: postgres
```

Comment:
why remove the env override? isn't no password the same as having an insecure one?

Reply:
I removed the env overrides so that other repos docker compose files have a consistent password they can use to connect to the database. This is now functionally the same as the rabbitmq instance, which uses `guest:guest` as the default. The postgres instance now uses `postgres:postgres` as the default.